### PR TITLE
Add Contact Page

### LINKED ISSUE #211: Add a Contact Us Page/Section
What if a passenger loses something or needs help with a schedule?  We should add a page for this, which lists the numbers for lost and found and the non-employee phone numbers for each garage, as well as the PVTA's main line.

I could see it going as its own page in Settings, or as a subsection in About.

### PR DESCRIPTION
**Pull request by akaplo**
Closes #211.  Adds a Contact page at `/about/contact` that gives a bunch of phone numbers for passengers.  Nothing special here.  @sherson, @werebus, @petermarathas, and @bgregg, any thoughts on UI or numbers/links that should or should not be included?

screen shot 2016-09-25 at 10 56 35 pm

### CODE REVIEW

File: www/pages/contact/contact.html [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+        Paratransit Van Requests
+      </div>
+      <div class="item item-button-right">
+        PVTA Paratransit Van Requests (All Areas):
+        <span class="item-note">413-732-6248, ext. 214</span>
+        <a class="button button-positive" href="tel:413-732-6248">
+          <i class="icon ion-android-call"></i>
+        </a>
+      </div>
+      <div class="item item-button-right">
```
Comment (werebus) [inline www/pages/contact/contact.html:82]: I'd lose SpecTrans.  It isn't really anything to do with the PVTA.

**Review by werebus — CHANGES_REQUESTED** [review]
Social media stuff?  PVTA does have a Twitter and Facebook page (with limited content)  So does UMTS.  I dunno.

**Comment by akaplo** [comment]
I like the social media stuff; will add.

**Comment by akaplo** [comment]
@werebus 
screen shot 2016-09-28 at 11 16 47 am

**Comment by werebus** [comment]
Icons are a little inconsistent.

**Comment by akaplo** [comment]
Sizes?  Yeah, I'm not sure why that is; they're supposed to all be the same.

**Comment by werebus** [comment]
Well, presumably it's not a fixed-width font.  Buttons with "text" on them are usually as wide as the text requires, right?  It's hard to tell, but it looks like they're all the same height.

**Comment by akaplo** [comment]
Sure, but the button doesn't have text; it has an icon.
Its only shown content is

`<i class="icon ion-social-facebook"></i>`

Either way, I'm working on seeing if I can change the size.

**Comment by akaplo** [comment]
Fixed with some `padding-left` for the Facebook icon's text.

**Pull request merged**